book/layout.md

```diff
@@ -188,7 +188,20 @@ The core idea is that we'll have a whole tree of `BlockLayout` objects
 that contain text, and they'll lay out their contents the way we've
 already implemented. But there will also be new, intermediate
 `BlockLayout`s with `BlockLayout` children, and they will stack their
-children vertically.
+children vertically. (An example is shown in Figure 1.)
```

Comment:
This figure and parenthetical is moved from much later in the chapter.

Reply:
How important is this?